src/GraphQL.Benchmarks/NullStream.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.IO;
+
+namespace GraphQL.Benchmarks
+{
+    internal class NullStream : Stream
```

Comment:
Stream.Null?

Reply:
When I wrote this, I was in a hurry and did not think about Stream.Null.

Reply:
Ok. With Stream.Null nothing changed.